__tests__/sumTest.js

```diff
@@ -0,0 +1,7 @@
+function sum(a, b) {
```

Comment:
all tests files should have the `.test.js` or `.spec.js` suffixes, and the folder can just be `test`

Reply:
Fixed it